IFC4x3/Properties/n/Na+_1yejeoaXj1gAsLahNn8Fv1/DocProperty.xml

```diff
@@ -0,0 +1,2 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="Na+_1yejeoaXj1gAsLahNn8Fv1" Name="Na+" UniqueId="7ca2da32-921b-41a8-ad95-92b5f120fe41" PrimaryDataType="IfcReal" />
```

Comment:
NatriumContent

Reply:
Keeping the names proposed by the domain experts.